fastlane_core/lib/fastlane_core/update_checker/update_checker.rb

```diff
@@ -123,9 +123,11 @@ def self.generate_fetch_url(gem_name)
     def self.ios_app_identifier(args)
       # args example: ["-a", "com.krausefx.app", "--team_id", "5AA97AAHK2"]
       args.each_with_index do |current, index|
+        # rubocop: disable Style/MultipleComparison
```

Comment:
I'm wondering if we should just disable this rule all together instead of on a case-by-case basis. IMO it's not a rule we want to enforce. I'm totally open to discussion on that though.